### ISSUE TITLE
List limit not working with Microsoft SQL Server

### ISSUE THREAD

**Issue by Erftralle** [joomla/joomla-cms#6414]
### Steps to reproduce the issue
Open the article manager in the backend and change the list limit (e.g. to 5, assuming you have more than 5 articles in your system)
### Expected result
The list should show only 5 items.
### Actual result
The list shows all items.
### System information (as much as possible)
Joomla! 3.4.0
Microsoft SQL Server 2012
### Additional comments
This is due to a change made in #5293.
An essential part of the query, namely
```
	$query = 'SELECT * FROM (' . $query . ') _myResults WHERE RowNumber BETWEEN ' . $start . ' AND ' . $end;
```
has been removed from the processLimit() function in **libraries/joomla/database/query/sqlsrv.php** (maybe by accident?).

**Comment by sovainfo** [comment]
Fixed by #7149

**Comment by zero-24** [comment]
Closing as we have a PR that fixes the issue thanks for checking @sovainfo and reporting @Erftralle
